### PR TITLE
Fixes #15033: Update backup and restore documentation 

### PR DESCRIPTION
**Pull request by peckpeck**
https://issues.rudder.io/issues/15033

### CODE REVIEW

File: src/reference/modules/administration/pages/procedures.adoc [hunk 1]
```diff
@@ -251,11 +251,12 @@ These procedures will explain how to backup your Rudder installation.
 
 === Backup
 
-This backup procedure will operate on the three principal Rudder data sources:
+This backup procedure will operate on principal Rudder data sources:
 
 * The LDAP database
 * The PostgreSQL database
 * The configuration-repository folder
+* Rudder cofiguration
```
Comment (amousset) [inline src/reference/modules/administration/pages/procedures.adoc]: configuration

File: src/reference/modules/administration/pages/procedures.adoc [hunk 2]
```diff
@@ -265,24 +266,26 @@ It will also backup the application logs.
...
 
-# Or without sudo, use the rudder application password:
-pg_dump -U rudder rudder > /tmp/rudder-backup-$(date +%Y%m%d).sql
+# Or without sudo, use the rudder application password:
+pg_dump -Fc -U rudder rudder > /tmp/rudder-backup-$(date +%Y%m%d).sql
 
-# Third, backup the configuration repository:
-tar -C /var/rudder -zvcf /tmp/rudder-backup-$(date +%Y%m%d).tar.gz configuration-repository/ cfengine-community/ppkeys/
+# Third, backup the configuration repository:
+tar -C /var/rudder -zcf /tmp/rudder-backup-$(date +%Y%m%d).tar.gz configuration-repository/ cfengine-community/ppkeys/ packages/
```
Comment (amousset) [inline src/reference/modules/administration/pages/procedures.adoc]: plugins folder too

Comment (peckpeck) [reply src/reference/modules/administration/pages/procedures.adoc]: packages is the plugin folder

**Comment by peckpeck** [comment]
Commit modified

File: src/reference/modules/plugins/pages/windows.adoc [hunk 3]
```diff
@@ -1,206 +0,0 @@
-[[dsc-plugin]]
```
Comment (amousset) [inline src/reference/modules/plugins/pages/windows.adoc]: why removing the windows doc?

**Comment by peckpeck** [comment]
Commit modified

File: src/reference/modules/administration/pages/procedures.adoc (hunk 2 above)
Comment (amousset) [reply src/reference/modules/administration/pages/procedures.adoc]: ok!

Comment (amousset) [reply src/reference/modules/administration/pages/procedures.adoc]: /var/rudder/plugin-resources contains plugins configuration

**Comment by peckpeck** [comment]
Commit modified

**Pull request merged**